apps/base/models.py

```diff
@@ -14,3 +19,28 @@ class TimeStampedModel(models.Model):
     class Meta:
         abstract = True
         app_label = 'base'
+
+
+"""
+The decorator below is used to pass model field names to django signals function.
+"""
+
+
+def extra_args(field_name, *args, **kwargs):
```

Comment:
One more thing, can we use some other function name. `extra_args` and its related functions - these names are fine, just that they dont give any impression of what they are meant for.